Utilities/gdcmcharls/lookuptable.h

```diff
@@ -45,7 +45,7 @@ class CTable
 
     CTable()
     {
-        ::memset(_rgtype, 0, sizeof(_rgtype));
+        ::memset((void*)_rgtype, 0, sizeof(_rgtype));

```

Comment:
Wait, this isn't fixing anything, it's just suppressing the warning... and the warning looks correct to me.